mmv1/templates/terraform/examples/gkeonprem_vmware_cluster_basic.tf.erb

```diff
@@ -0,0 +1,43 @@
+resource "google_gkeonprem_vmware_cluster" "<%= ctx[:primary_resource_id] %>" {
+  name = "cluster-<%= ctx[:vars]['name'] %>"
+  location = "us-west1"
+  admin_cluster_membership = "projects/870316890899/locations/global/memberships/gkeonprem-terraform-test"
+  description = "test cluster"
+  on_prem_version = "1.13.1-gke.35"
+  annotations = {}
+  network_config {
+    service_address_cidr_blocks = ["10.96.0.0/12"]
+    pod_address_cidr_blocks = ["192.168.0.0/16"]
+    dhcp_ip_config {
+      enabled = true
+    }
+  }
+  control_plane_node {
+     cpus = 4
+     memory = 8192
+     replicas = 1
+  }
+  load_balancer {
+    vip_config {
+      control_plane_vip = "10.251.133.5"
+      ingress_vip = "10.251.135.19"
+    }
+    metal_lb_config {
+      address_pools {
+        pool = "ingress-ip"
+        manual_assign = "true"
+        addresses = ["10.251.135.19"]
+        avoid_buggy_ips = true
+      }
+      address_pools {
+        pool = "lb-test-ip"
+        manual_assign = "true"
+        addresses = ["10.251.135.19"]
+        avoid_buggy_ips = true
+      }
+    }
+  }
+
+  provider = google-beta
```

Comment:
nit: for this + other examples do you mind cleaning up the extra newlines + putting this on the second line of the example? i.e.

```
resource "google_gkeonprem_vmware_cluster" "<%= ctx[:primary_resource_id] %>" {
  provider = google-beta

  name = "cluster-<%= ctx[:vars]['name'] %>"
...
```

Reply:
Updated in all examples. Thanks for the suggestion